app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: None) -> None:
+    func_results = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
You could limit processing to `args` only, there is no need to process `kwargs` for this task.

Reply:
done